crypto/conf/conf_def.c

```diff
@@ -123,7 +123,6 @@ static int def_init_default(CONF *conf)
 
     conf->meth = &default_method;
     conf->meth_data = (void *)CONF_type_default;
-    conf->data = NULL;
```

Comment:
Why?

Reply:
Because now we're using zalloc they're already initialised to NULL.

Reply:
So this is only called directly after def_new()?

Reply:
> So this is only called directly after def_new()?

Ah! You're right! It might be called through other routes. Fixup pushed.